web/styles/zulip.css

```diff
@@ -1294,17 +1294,6 @@ label {
     line-height: inherit;
 }
 
-/* Bootstrap's focus outline uses -webkit-focus-ring-color which doesn't exist in Firefox */
```

Comment:
While I can't wait to make these styles go away, this is way too soon. Your patches above only fix the compose area, which is great--we want to go in slow, measured stages. But for the time being, you need to ensure that your fixes in `compose.css` work without having to remove these styles just yet.